ChangeLog

```diff
@@ -1,3 +1,11 @@
+2015-05-27  Jim Shaver <dcypherd@gmail.com>
+
+	* OpenSSL/SSL.py, : Add ``get_protocol_version()`` and 
+	  ``get_protocol_version_name()`` to Connection
+	  Based on work from Rich Moore
+	* OpenSSL/test/test_crypto.py: tests for ``get_protocol_version()``
```

Comment:
just add  OpenSSL/test/test_crypto.py to the list above.  it’s implicit that we add tests for new code. :)